README.md

```diff
@@ -123,6 +124,8 @@ Copy all the files from `Source` folder.
 - OpenIconic.ttf
 - Stateface.ttf
 - WeatherIcons.ttf
+- WeatherIcons.ttf
```

Comment:
Can you remove this line? `WeatherIcons` is listed twice here. Thanks.

Reply:
Oops! yes sure.
Thanks for noticing!

Reply:
Done!